src/components/Observations/GridItem.js

```diff
@@ -79,7 +80,13 @@ const GridItem = ( {
             />
           </View>
         )}
-        <ObsCardStats item={item} view="grid" />
+        {!item._synced_at
```

Comment:
Observations that were synced but have since been changed locally also need to be uploaded, so you might want to replace this with a model method. We have `wasSynced()`, but maybe we need `needsSync()`.